README.md

```diff
@@ -1,2 +1,15 @@
-# apple-clone
-Microverse Project: Building with Backgrounds and Gradients
+# Apple Clone
+
+## Cloning [Archived Apple page](https://web.archive.org/web/20140301004610/http://www.apple.com/), a Microverse Project
+
+[Demo](https://rawcdn.githack.com/fedgut/apple-clone/70e3a0818559e7eb74fe7d1aa9bf26478801dd43/index.html)
+
+# Built With
+- HTML
+- CSS
+
+# Author
+- [Eduardo Gutierrez](https://github.com/fedgut)
+
+# Acknowledgments
+- [Microverse](https://microverse.org)
```

Comment:
Work on improving the readme file, so you start getting used to giving a good professional image to your repositories. You have a[ free course here ](https://www.udacity.com/course/writing-readmes--ud777
)and a good template to find inspiration [here](https://github.com/othneildrew/Best-README-Template).